spec/rubocop/cop/style/redundant_line_continuation_spec.rb

```diff
@@ -192,6 +192,22 @@ def self.foo(bar,#{trailing_whitespace}
     RUBY
   end
 
+  it 'does not register an offense when line continuations with using && for comparison chaining' do
+    expect_no_offenses(<<~'RUBY')
+      foo == other.foo \
+        && bar == other.bar \
+        &&  baz == other.baz
```

Comment:
Is this a typo?

```suggestion
        && baz == other.baz
```

Reply:
Oh, I'm sorry. I missed that redundant whitespace. Thank you for pointing it out. I updated this PR.